src/test/kotlin/wordle/domain/WordTest.kt

```diff
@@ -0,0 +1,72 @@
+package wordle.domain
+
+import org.assertj.core.api.Assertions.assertThat
+import org.junit.jupiter.api.Test
+import org.junit.jupiter.api.assertThrows
+import org.junit.jupiter.params.ParameterizedTest
+import org.junit.jupiter.params.provider.ValueSource
+
+class WordTest {
+
+    @Test
+    fun `단어를 만들 수 있다`() {
+        val givenWord = "value"
+        val word = Word(givenWord)
+
+        assertThat(word.value).isEqualTo(givenWord)
+    }
+
+    @ParameterizedTest
+    @ValueSource(strings = ["abc", "abcdef"])
+    fun `단어는 5글자가 아닌 경우 예외를 던진다`(wrongWord: String) {
+        val message = assertThrows<IllegalArgumentException> {
+            Word(wrongWord)
+        }.message
+
+        assertThat(message).isEqualTo("5글자여야합니다")
```

Comment:
```suggestion
        assertThatIllegalArgumentException().isThrownBy { Word(wrongWord) }.withMessageContaining("5글자여야합니다")
```

IllegalArgumentException에 대한 테스트에 대해서 이렇게 표현할 수도 있을 것 같습니다.